Dockerfile.armhf

```diff
@@ -65,9 +65,7 @@ RUN cd /usr/local/lvm2 \
 # see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
 
 # Install Go
-# TODO Update to 1.5.4 once available, or build from source, as these builds
-# are marked "end of life", see http://dave.cheney.net/unofficial-arm-tarballs
-ENV GO_VERSION 1.5.3
+ENV GO_VERSION 1.6.2
 RUN curl -fsSL "http://dave.cheney.net/paste/go${GO_VERSION}.linux-arm.tar.gz" \
```

Comment:
This won't work; these builds are no longer updated, but, official builds should now be available :tada: see https://github.com/docker/docker/pull/21987#issue-147998486, and https://twitter.com/davecheney/status/720180308675403776

> With the release of Go 1.6 the Go team have made available official ARMv61 binary releases. Thus, this page is now end of life and will no longer be updated. The existing Go 1.4.x and 1.5.x releases will remain for a time to assist people bootstrapping ARM systems.
> For other arm variants, I recommend cross compiling or building from source:
> Build from source, http://dave.cheney.net/2015/09/04/building-go-1-5-on-the-raspberry-pi
> Cross compile, http://dave.cheney.net/2015/08/22/cross-compilation-with-go-1-5
> Notes
>
>ARMv6 is upwards compatible with ARMv7 systems. There is no material performance difference running an ARMv6 binary on an ARMv7 system.

I'm very bad at these, so not sure if these; https://storage.googleapis.com/golang/go1.6.2.linux-armv6l.tar.gz are compatible with armhf, or that we need to compile from source?